Board Meeting Minutes/CCC Board Minutes 2020-01-16.md

```diff
@@ -0,0 +1,66 @@
+**Governing Board Conference Call -- 7:00am PST**
+
+**16 January 2020**
+
+1.  **Call to Order / Roll Call**
+
+    a.  **Premier Members of the CCC**
+
+        i.  Philippe Robin (Arm)\*
+
+        ii. Sergio Leunissen (Oracle)\*
+
+        iii. Mike Bursell (Red Hat)\*
+
+        iv. Stephen Walli (Microsoft)\* Chair
+
+        v.  Ron Perez (Intel)*
+
+> **Guests / Observers**
+
+vi. Dave Thaler (Microsoft) (TAC Chair)
```

Comment:
I believe this is incorrect.  Per the CCC charter, the TAC chair is a Member of the CCC, not a Guest/Observer.

Since these minutes were approved, I guess it's ok to post as is, but I'd like to log that the minutes are incorrect and request they be corrected.